packages/suite-desktop-core/e2e/support/common.ts

```diff
@@ -105,10 +105,15 @@ export const launchSuite = async (params: LaunchSuiteParams = {}) => {
     return { electronApp, window };
 };
 
+export const isDesktopProject = (testInfo: TestInfo) =>
```

Comment:
I originally wanted to condition the visual comparison by this boolean. but then I found out you can add the --ignore-snapshot to project definition. But anyway I used this new methods on few places so I would keep it as a minor unrelated refactoring.